central/notifiers/jira/jira.go

```diff
@@ -92,18 +113,29 @@ func getIssueTypes(client *jiraLib.Client, project string) ([]*jiraLib.MetaIssue
 		return []*jiraLib.MetaIssueType{}, err
 	}
 
-	result := &issueTypeResult{}
+	if isCloud == nil {
+		isCloud = func() bool {
+			return strings.Contains(client.GetBaseURL().Host, "atlassian.net")
```

Comment:
Custom domains are in beta now so this is a short term workaround https://community.atlassian.com/t5/Enterprise-articles/Custom-Domains-in-Cloud-Development-Update/ba-p/2283483

Reply:
I'll see if there is a better way to determine this before merge.  It would be really great to avoid having to fix this yet again based on customer cases.

Reply:
I resolved this issue by having the same code path handle responses from both Jira server and cloud.  The only difference turned out to be that the name of the key in the response JSON that contained the list of items.